File: regressproof/fixtures/preexisting-js/tracked/current/scripts/lint.js
const fs = require("node:fs");

const source = fs.readFileSync("src/app.js", "utf8");

if (source.includes("LINT_FAIL")) {
  console.error("lint failed in src/app.js: found LINT_FAIL marker");
  process.exit(1);
}

console.log("lint passed");

File: regressproof/fixtures/swift-js/scripts/lint.swift
import Foundation

let source = try String(contentsOfFile: "src/App.swift", encoding: .utf8)

if source.contains("LINT_FAIL") {
    FileHandle.standardError.write(Data("lint failed in src/App.swift: found LINT_FAIL marker\n".utf8))
    exit(1)
}

print("lint passed")
